Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -165,6 +178,33 @@ void DMainWindow::OnOpen()
 		StartGame(filename);
 }
 
+void DMainWindow::OnBrowse()
+{
+	std::string path = ShowFolderDialog().toStdString();
+	std::vector<std::string>& iso_folder = SConfig::GetInstance().m_ISOFolder;
+	if (!path.empty())
+	{
+		auto itResult = std::find(
+				iso_folder.begin(),
+				iso_folder.end(), path);
```

Comment:
Probably a single line is best, now that it's not doing the whole GetInstance shebang for the iso folder vector.